osgp/platform/osgp-adapter-ws-tariffswitching/src/main/java/org/opensmartgridplatform/adapter/ws/tariffswitching/application/mapping/ws/TariffScheduleToScheduleConverter.java

```diff
@@ -40,26 +38,21 @@ public ScheduleEntry convert(final TariffSchedule source, final Type<? extends S
         schedule.setIsEnabled(source.isIsEnabled());
         schedule.setMinimumLightsOn(source.getMinimumLightsOn());
 
-        try {
-            // Set the lightvalue
-            // For now a High tariff means the Relay is switched off (Situation
-            // in Zaltbommel)
-            // schedule.setLightValue( Arrays.asList(new LightValue(null,
-            // !source.isHigh(), null)) )
-
-            final List<LightValue> lightValues = new ArrayList<>();
-            for (final TariffValue tariffValue : source.getTariffValue()) {
+        // Set the lightvalue
+        // For now a High tariff means the Relay is switched off (Situation
+        // in Zaltbommel)
+        // schedule.setLightValue( Arrays.asList(new LightValue(null,
+        // !source.isHigh(), null)) )
```

Comment:
I see this was only a formatting change. I wouldn't have complained if these lines would have been deleted, though.
Just noticed the switch from tariff to light value. Some comment might have been helpful. Safe to ignore this further as far as the get status changes are concerned.

Only wondering why this code was touched at all, but that might be already explained looking at the validation exception that was removed from the `LightValue` constructor.